BEIMA.Backend/MongoService/MongoConnector.cs

```diff
@@ -45,6 +46,24 @@ private MongoConnector()
                 credentials = Environment.GetEnvironmentVariable("AzureCosmosConnection");
             }
             client = new MongoClient(credentials);
+
+            // Create default admin user on startup if no admin user exists.
+            //
+            // The credentials for this default admin user are:
+            // Username = first.admin
+            // Password = Abcdefg12345!
+            //
+            // !!!!!!!!! IMPORTANT, PLEASE READ !!!!!!!!!
+            // It is highly encouraged that a new, more secure admin user is set up in place of this default admin user
+            // once the application has been successfully set up. The purpose of this user is to provide a means to create
+            // users when the application is launched for the first time. This default user is also used for testing.
+            if (!(GetFilteredUsers(MongoFilterGenerator.GetEqualsFilter("role", Constants.ADMIN_ROLE)).Count > 0))
```

Comment:
Just wanted to mark this spot to make sure you guys see this. This is how we are going to be setting the default user on startup.